temporalcli/commandsmd/commands.md

```diff
@@ -1,487 +1,1185 @@
+### temporal activity fail: Fail an Activity
 
-Fail an Activity.
+Fail an Activity, marking it as having encountered an error. Specify the
+Activity and Workflow IDs:
 
-`temporal activity fail --activity-id=MyActivityId --workflow-id=MyWorkflowId`
+```
+temporal activity fail \
+    --activity-id YourActivityId \
+    --workflow-id YourWorkflowId
+```
 
 #### Options
 
-* `--activity-id` (string) - The Activity to be failed. Required.
-* `--detail` (string) - JSON data describing reason for failing the Activity.
-* `--identity` (string) - Identity of user submitting this request.
-* `--reason` (string) - Reason for failing the Activity.
+* `--activity-id` (string) -
+  `ID` of the Activity to be failed.
+  Required.
+* `--detail` (string) -
+  JSON data with the reason for failing the Activity.
+* `--identity` (string) -
+  Identity of the user submitting this request.
+* `--reason` (string) -
+  Reason for failing the Activity.
 
 Includes options set for [workflow reference](#options-set-for-workflow-reference).
 
+### temporal batch: Manage batch jobs
+
+
+A batch job executes a command affecting multiple Workflow Executions at once.
+Specify the Workflow Executions to include and the type of batch job to apply.
+For example, to cancel all running 'YourWorkflow' Workflows:
+
+```
+temporal batch workflow cancel \
+  --query 'ExecutionStatus = "Running" AND WorkflowType="YourWorkflow"' \
+  --reason "Testing"
+```
 
-### temporal batch: Manage Batch Jobs
+The `batch` command keyword is optional when you supply a `--query`. This
+invocation is identical to the previous one:
 
-Batch commands change multiple Workflow Executions.
+```
+temporal workflow cancel \
+  --query 'ExecutionStatus = "Running" AND WorkflowType="YourWorkflow"' \
+  --reason "Testing"
+```
 
 #### Options
 
 Includes options set for [client](#options-set-for-client).
 
-### temporal batch describe: Show Batch Job progress.
+### temporal batch describe: Show batch job progress
 
-The temporal batch describe command shows the progress of an ongoing Batch Job.
+Show the progress of an ongoing batch job. Pass a valid job ID to display
+its information:
 
-`temporal batch describe --job-id=MyJobId`
+```
+temporal batch describe --job-id YourJobId
+```
 
 #### Options
 
-* `--job-id` (string) - The Batch Job Id to describe. Required.
+* `--job-id` (string) -
+  Batch job ID.
+  Required.
 
-### temporal batch list: List all Batch Jobs
+### temporal batch list: List all batch jobs
 
-The temporal batch list command returns all Batch Jobs.
-Batch Jobs can be returned for an entire Cluster or a single Namespace.
-
-`temporal batch list --namespace=MyNamespace`
+Return a list of batch jobs on the Service or within a single Namespace.
+For example, list the batch jobs for "YourNamespace":
+```
+temporal batch list --namespace YourNamespace
+```
 
 #### Options
 
-* `--limit` (int) - Limit the number of items to print.
+* `--limit` (int) -
+  Max output count.
```

Comment:
This seems a little obtuse—"count" of what?

Reply:
I've been struggling with this a lot. I'll kick it around and workshop something better. Thanks!